src/libutil/util.cc

```diff
@@ -1025,6 +1026,16 @@ void runProgram2(const RunOptions & options)
         if (source && dup2(in.readSide.get(), STDIN_FILENO) == -1)
             throw SysError("dupping stdin");
 
+        //if (options.chdir && chdir((*options.chdir).c_str()) == -1)
+        //    throw SysError("chdir failed");
```

Comment:
Was this intended to be commented out?

Reply:
Oops ... Commented that out thinking it was causing #2801. I'll uncomment and re-test. Good catch, thank you.